documentation/docs/30-advanced/67-shallow-routing.md

```diff
@@ -93,6 +93,6 @@ For this to work, you need to load the data that the `+page.svelte` expects. A c
 
 ## Caveats
 
-During server-side rendering, `$page.state` is always an empty object. The same is true for the first page the user lands on — if the user reloads the page, state will _not_ be applied until they navigate.
+During server-side rendering, `$page.state` is always an empty object. The same is true for the first page the user lands on — if the user reloads the page (or comes back from another domain), state will _not_ be applied until they navigate.
```

Comment:
I wonder whether this should technically say origin rather than domain. I'm not sure which it is

Reply:
it's actually neither — it's about whether it's a cross-_document_ navigation

```suggestion
During server-side rendering, `$page.state` is always an empty object. The same is true for the first page the user lands on — if the user reloads the page (or comes back from another document), state will _not_ be applied until they navigate.
```